dm/tests/check_task/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
maybe move it into dmctl_basic/*/check_task.sh

Reply:
Task checking isn't solely relevant to `dmctl` and it has incurred a noticeable number of issues (including connection checking etc.). So I'm planning to decouple these test cases from others and test them as a whole.